quickwit-config/src/index_config.rs

```diff
@@ -320,6 +321,18 @@ impl RetentionPolicy {
         })
     }
 
+    pub fn duration_till_next_evaluation(&self) -> anyhow::Result<Duration> {
+        let schedule = self.evaluation_schedule()?;
+        let future_date = schedule
+            .upcoming(Utc)
+            .next()
+            .context("Failed to obtain next evaluation date.")?;
```

Comment:
Having to deal with an option is an artifact of the iterator interface. Can a cron expression be "finite"? Shouldn't this be an `expect`?